### PR TITLE
Added works and about

### PR DESCRIPTION
**Pull request by bekjon-me**
- added works section(portfolio section)
- added about myself section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,5 +56,175 @@ <h1>
...
+            A daily selection of privately <br />
+            personalized reads; no accounts or <br />
+            sign-ups required.
+          </p>
+          <ul class="tags">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button type="button">See Project</button>
```
Comment (mwanawabangona) [inline index.html:82]: - [ ] `required`
- [ ] please kindly add a hover to the button

File: index.html [hunk 2]
```diff
@@ -56,5 +56,175 @@ <h1>
...
```
Comment (mwanawabangona) [inline index.html:109]: - [ ] `required`
- [ ] please kindly add a hover to the button

File: index.html [hunk 3]
```diff
@@ -56,5 +56,175 @@ <h1>
...
```
Comment (mwanawabangona) [inline index.html:133]: - [ ] `required`
- [ ] please kindly add a hover to the button

File: index.html [hunk 4]
```diff
@@ -56,5 +56,175 @@ <h1>
...
```
Comment (mwanawabangona) [inline index.html:160]: - [ ] `required`
- [ ] please kindly add a hover to the button

File: index.html [hunk 5]
```diff
@@ -56,5 +56,175 @@ <h1>
...
+          /></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li>
+          <a href="#"><img src="./src/assets/Angelina.svg" alt="" /></a>
+        </li>
+      </ul>
+      <button type="button">Get my resume</button>
```
Comment (mwanawabangona) [inline index.html:195]: - [ ] `required`
- [ ] please kindly add a hover to the button

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
## Status: Changes Required

 ## Hello there @bekjon-me  👋🏽

###  You have done an amazing job so far! 👏🏽 👏🏽


There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand:


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Solid pull request scroll
* ✅  Good use of flexbox 🧐
* ✅  Good quality commits

## Required Changes
 _**Check the comments under the review.**_

 ## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @mwanawabangona in your question so I can receive the notification.
>
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
>
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @bekjon-me , 
Good job so far!

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSE
 

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**